Sources/Swift/Integrations/UserFeedback/Configuration/SentryUserFeedbackConfiguration.swift

```diff
@@ -53,12 +53,6 @@ public class SentryUserFeedbackConfiguration: NSObject {
      */
     public var tags: [String: Any]?
     
-    /**
-     * Sets the email and name field text content to `SentryUser.email` and `SentryUser.name`.
-     * - note: Default: `true`
-     */
-    public var useSentryUser: Bool = true
```

Comment:
this is only used in the form, so i figured it made more sense in SentryUserFeedbackFormConfiguration

Reply:
Doesn't it also make sense in the `captureFeedback` API's direct use? Like I can skip passing the user if I know it's on the scope with this option.

Reply:
hmm good question. so as far as these configuration classes are concerned, they shouldn't have any bearing on how that captureFeedback behaves, only the widget and form.

but, i can see how it would be useful to grab the user from the scope inside that function. it's not something that's done today. let me take a look at adding that here.

Reply:
@philipphofmann i'm not sure of any way to access the current user from this swift implementation which will be public facing, since SentryScope.userObject is declared in SentryScope+Private.h :/

Reply:
We can make `SentryScope.userObject` public, but I think we should name it `SentryScope.getUser` to align with `SentryScope.setUser`. It would be great to do this in an extra PR cause it deserves a changelog entry.
We didn't make it public cause of synchronization issues, I guess. The method must return a copy of the user object and clearly state this, because we quite often read from the user object on BG threads when applying the scope, unless you have a better idea to avoid synchronization issues.

Reply:
Yes, I agree with doing it separately, I meant to call this out last night before you reviewed. Made a task for it: https://github.com/getsentry/sentry-cocoa/issues/4884